src/include/uservarmanager.h

```diff
@@ -14,6 +14,15 @@
     #include "globals.h"
 #endif
 
+class UserVarManagerUI
```

Comment:
Doxygen comments depicting the interface of these.

Reply:
in a6f3a04